src/Chirp.WebService/Pages/Public.cshtml.cs

```diff
@@ -43,10 +45,16 @@ public ActionResult OnGet()
         }
         
         Cheeps = _cheepRepository.GetCheepsForPage(PageNumber);
-        
         //Set the follows
         Follows = _authorRepository.GetFollowsForAuthor(User.GetUserEmail());
         
         return Page();
     }
+
+    public bool CheepIsLiked(Guid cheepId)
+    {
+        var authorId = User.GetUserId() ?? Guid.Empty;
```

Comment:
If we know the Guid is empty, we should probably return false immediately before looking through the databse.